### PR TITLE
fix: show balances for every coin in the token search

### PR DESCRIPTION
**Pull request by davidkoller318**
issue #15

### CODE REVIEW

**Comment by chillvybz** [comment]
Maybe don't display anything when in the wallet dis-connected state:
image

**Comment by chillvybz** [comment]
After I connect the wallet, the list doesn't refresh until I refresh the browser:
image

**Comment by chillvybz** [comment]
Can we also move the Balance number underneath the Symbol like how the main UI shows?
image

No need for "Bal =" however, just the amount is fine.

**Comment by davidkoller318** [comment]
@chillvybz 
I fixed the above issues you mentioned.

File: src/client/js/components/partials/TokenSearchBar.jsx [hunk 1]
```diff
@@ -113,6 +132,19 @@ export default class TokenSearchBar extends Component {
     var rest = _.rest(this.TOP_TOKENS, 3);
 
     var top3Content = _.map(top3, function(v, i) {
+
+      const tokenBalance = this.state.tokenBalances[v.symbol];
+      let balanceNumber = null;
+      if (tokenBalance) {
```
Comment (chillvybz) [inline src/client/js/components/partials/TokenSearchBar.jsx:138]: This code get repeated quite a bit:
can we standardize it into a helper method.
I know it's used in TokenSymbolBalance.jsx as well

**Comment by davidkoller318** [comment]
@chillvybz 
I refactored the repetitive code and turned it into a helper method.

**Review by chillvybz — APPROVED** [review]

**Pull request merged**